package camPackage;
import "gps.proto";
import "obd2.proto";

message CAM {
	required string stationId = 1;
	required int32 id = 2;
	required string content = 3;
	optional int64 createTime = 4;
	optional gpsPackage.GPS gps = 5;
	optional obd2Package.OBD2 obd2 = 6;
	optional double heading = 7;
}
